django-rest/imdb_app/users/serializers.py

```diff
@@ -18,6 +18,22 @@ def create(self, validated_data):
         user.save()
         return user
 
+
+    def update(self, instance, validated_data):
+        
+        if 'first_name' in validated_data:
```

Comment:
Sto bi sam hendlao za vecinu ovih? To bi trebao da hendla sam django na PATCH ruti, aj password verovatno ne, ali set password zavisi ako radis change password da ne moze da izmeni ako je isti kao prosli na primer

Reply:
Ja bih obrisao vecinu ovoga i ostavio da Django radi na patch ruti, jedino mozes password da ostavis ovde ako ima potrebe, ali ako bi imao druge validacije onda bi bila druga ruta za password